src/nat/middleware/defense_middleware_content_guard.py

```
@@ -0,0 +1,467 @@
+                # Redirected: yield replacement once (and stop).
+                yield processed_output
+                return
+
+            if buffer_chunks:
+                # refusal: would have raised; safe content: preserve chunking
+                for chunk in accumulated_chunks:
+                    yield chunk
+
+        except Exception as e:
+            logger.exception(
+                "Failed to apply content safety guard to streaming function %s: %s", context.name, e
```

Ruff (0.14.8)

Redundant exception object included in `logging.exception` call

(TRY401)